src/ext.js

```diff
@@ -26,7 +27,11 @@ const toggleOptions = [
   },
 ];
 
+
+
 export default function ext({ translator, shouldHide, senseNavigation }) {
+  const stylingPanelEnabled = shouldHide.isEnabled && shouldHide.isEnabled('SENSECLIENT_IM_1525_BUTTON');
+  const bkgOptionsEnabled = shouldHide.isEnabled && shouldHide.isEnabled('BUTTON_BKG_OPTIONS');
```

Comment:
Is this feature flag new? Does it belong to the same improvement? In that case, add the `IM_1525` to the flag name

Reply:
This feature flag is used for the same purpose like we did back in KPI: to enable/disable the background options on styling panel for this chart and the KPI uses the flag `KPI_BKG_COLOR`, so we thought of maintaining the same name consistency only renaming the `COLOR` part to `OPTIONS`, also the background options belong to another IM i.e IM-323.